src/api/app/views/webui/monitor/_building_table.html.haml

```diff
@@ -53,6 +53,3 @@
         'bPaginate': false,
         'oSearch': {'sSearch': '#{@project_filter}', 'bRegex': false, 'bSmart': false},
       });
-- else
```

Comment:
I think the else case will never be called. Somewhere else the output "All xxx hosts are idle!" will be generated and put into the page.